### PR TITLE
Cart reducer

### PR DESCRIPTION
**Pull request by Mohitk391**
Added Cart Reducer
Files to check: 
1) CartReducer.js
2) Cart.js

preview: https://62d517975eb1c47fdf2403b9--optimistic-noether-0b2e72.netlify.app/

### CODE REVIEW

File: src/pages/cart/cart.css [hunk 1]
```diff
@@ -1,81 +1,107 @@
...
     text-align: center;
-    font-size: 1.2rem;
-}
-.card-quantity i {
-    cursor: pointer;
 }
 
 .cart-price-content{
-    display: flex;
+    width: 18em;
```
Comment (MayankKumar10) [inline src/pages/cart/cart.css]: use rem instead of em

File: src/pages/cart/cart.css [hunk 2]
```diff
@@ -1,81 +1,107 @@
...
-    border:none;
-    color: var(--clr-white);
-    border-radius: 1rem;
-    background-color: var(--cart-blue);
-    cursor: pointer;
+
+
+/*
+
+.main {
```
Comment (MayankKumar10) [inline src/pages/cart/cart.css]: remove commented code

**Review by MayankKumar10 — APPROVED** [review]
Look into these Suggestion everything else Looks Good to me you can merge it

**Review by Ankur9669 — APPROVED** [review]
Apart from changes suggested by mayank code looks fine

File: src/pages/cart/cart.css (hunk 1 above)
Comment (Mohitk391) [reply src/pages/cart/cart.css]: Thanks for the suggestion. I used em for relative responsiveness.

**Pull request merged**